documentation/modules/ROOT/pages/configuration/event-flattening.adoc

```diff
@@ -265,6 +265,8 @@ If you are configuring the event flattening SMT on a sink connector, setting thi
 |
 |Set this option to a comma-separated list, with no spaces, of metadata fields to add to the simplified Kafka record's value. When there are duplicate field names, to add metadata for one of those fields, specify the struct as well as the field, for example `source.ts_ms`. +
  +
+Optionally one can map with the new field name like `version:VERSION, connector:CONNECTOR, source.ts_ms:EVENT_TIMESTAMP` +
```

Comment:
```suggestion
Optionally, you can override the field name via `<field name>:<new field name>`, e.g. like so: new field name like `version:VERSION, connector:CONNECTOR, source.ts_ms:EVENT_TIMESTAMP` +
```

Also I think it's a bit misleading that all mapped names are upper case, this might create the suggestion that it *must* be that way.

Reply:
Agreed.
I am adding

Optionally, you can override the field name via `<field name>:<new field name>`, e.g. like so: new field name like `version:VERSION, connector:CONNECTOR, source.ts_ms:EVENT_TIMESTAMP`. Please note that the `new field name` is case-insensitive.